### ISSUE TITLE
US cell 57 has mix of integer and FP values

### ISSUE THREAD

**Issue by danielskatz** [softwaresaved/international-survey#164]
Cell 57 of https://github.com/softwaresaved/international-survey/blob/master/analysis/results_us_2017_narrative.ipynb has integer values in some columns and floating points values in others.  These are all counts, I think, which should all be integers.

**Comment by Oliph** [comment]
Issue similar to #163.

**Comment by danielskatz** [comment]
yes, but not consistent, even as the results of a single question...

**Comment by Oliph** [comment]
Yes, I just linked the two issues to remind me the inconsistency.
